src/textarea.js

```diff
@@ -19,15 +19,13 @@ const CALLBACK_METHODS = ['onInput', 'onKeydown', 'onKeyup'];
  */
 export default class Textarea extends Editor {
   el: HTMLTextAreaElement;
-  isIE9: boolean;
 
   /**
    * @param {HTMLTextAreaElement} el
    */
   constructor(el: HTMLTextAreaElement) {
     super();
     this.el = el;
-    this.isIE9 = getIEVersion() === 9;
```

Comment:
`getIEVersion` function is only used here, so please remove the function itself. 🙏